src/docs/Components/Hero/code.mdx

```diff
@@ -79,6 +79,8 @@ The title of your component must be a [Mozaic's Heading](/Foundations/Typography
 </h1>
 ```
 
```

Comment:
Can we change the title `Details of elements` to 
Elements detail

Reply:
fixed